pkg/pgtune/misc_test.go

```diff
@@ -7,6 +7,37 @@ import (
 	"github.com/timescale/timescaledb-tune/internal/parse"
 )
 
+var memLevels = []uint64{
+	7 * parse.Gigabyte,
+	8 * parse.Gigabyte,
+	15 * parse.Gigabyte,
+	16 * parse.Gigabyte,
+	24 * parse.Gigabyte,
+	32 * parse.Gigabyte,
+	80 * parse.Gigabyte,
+}
+
+var memSettingsMatrix = map[uint64]map[string]string{}
+
+func init() {
```

Comment:
I'm not married to this initialization method, I can see the potential of getting out of sync was `memLevels`. Maybe a this init of `memSettingsMatrix` is more maintainable?:

```
var memSettingsMatrix = map[uint64]map[string]string{
    7 * parse.Gigabyte: map[string]string{MaxLocksPerTx: maxLocksValue[0]},
    8 * parse.Gigabyte: map[string]string{MaxLocksPerTx: maxLocksValue[1]},
    ...
}
```

And then the `init()` function just is the `for` loop and nothing else?

Reply:
Yeah I agree that declaring memSettingsMatrix outside of the `init()` loop makes sense for maintainability and because it's not really dynamically derived from anything so can just function as a static var like memLevels.